curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-023.md

```diff
@@ -17,10 +17,18 @@ In your style sheet, comment out the the line containing the `background-color`
 
 # --hints--
 
-Test 1
+You should comment out the `background-color: burlywood;` line in your CSS.
 
 ```js
+assert(code.match(/\/\*\s*background-color:\s*burlywood;?\s*\*\//i));
```

Comment:
Do we need to add the flag to not remove comments for this challenge? I forget if that applies to HTML/CSS

Reply:
Might as well put it in there just to be safe.